foruma-frontend/src/stories/Button.stories.js

```diff
@@ -0,0 +1,45 @@
+import MyButton from './Button.vue';
+
+export default {
+  title: 'Example/Button',
```

Comment:
Codacy has a fix for the issue: [Replace `'Example/Button'` with `"Example/Button"`](https://app.codacy.com/gh/agabani/foruma/pullRequest?prid=7388752)
```suggestion
  title: "Example/Button",
```